packages/aws-serverless/src/integration/awslambda.ts

```diff
@@ -1,20 +1,42 @@
 import { AwsLambdaInstrumentation } from '@opentelemetry/instrumentation-aws-lambda';
 import { SEMANTIC_ATTRIBUTE_SENTRY_OP, SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN, defineIntegration } from '@sentry/core';
-import { addOpenTelemetryInstrumentation } from '@sentry/node';
+import { generateInstrumentOnce } from '@sentry/node';
 import type { IntegrationFn } from '@sentry/types';
+import { eventContextExtractor } from '../utils';
 
-const _awsLambdaIntegration = (() => {
+interface AwsLambdaOptions {
+  /**
+   * Disables the AWS context propagation and instead uses
+   * Sentry's context. This should usually be `true` when
+   * using Sentry to instrument Lambdas.
```

Comment:
```suggestion
   * Sentry's context. Defaults to `true`, in order for Sentry trace propagation to take precedence,
   * but can be disabled if you want to AWS propagation to take precedence.
```

or something like this?

Reply:
Updated :)